storage/migrations/26_runtime_events_nft_transfers.up.sql

```diff
@@ -0,0 +1,10 @@
+BEGIN;
+
+-- CREATE INDEX ix_runtime_events_nft_transfers
```

Comment:
:smiling_face_with_tear:

Reply:
hadn't tried with this actually

Reply:
Not sure which part the "hadn't tried" refers to. But really the part I'm most curious about is:
How fast/slow is the query without the index, i.e. as-is?

Reply:
aha. I have been able to observe it being used on staging mainnet

without index:
```
                                                                                                                                                                     QUERY PLAN                                                                                                                                                                      
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 Limit  (cost=7715455.36..7715455.37 rows=1 width=353)
   ->  Sort  (cost=7715455.36..7715455.37 rows=1 width=353)
         Sort Key: round DESC, tx_index, ((body)::text)
         ->  Index Scan Backward using ix_runtime_events_evm_log_signature on runtime_events  (cost=0.57..7715455.35 rows=1 width=353)
               Index Cond: ((runtime = 'sapphire'::runtime) AND (evm_log_signature = '\xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef'::bytea))
               Filter: ((type = 'evm.log'::text) AND (jsonb_array_length((body -> 'topics'::text)) = 4) AND (((body -> 'topics'::text) ->> 3) = 'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAOQ='::text) AND ((body ->> 'address'::text) = encode(eth_preimage(('oasis1qpwe6h0hw6s0x4skujhznld065300p83wgxnvu0n'::text)::oasis_addr), 'base64'::text)))

```

with index:
```
                                                                           QUERY PLAN                                                                            
-----------------------------------------------------------------------------------------------------------------------------------------------------------------
 Limit  (cost=690.69..690.70 rows=1 width=353)
   ->  Sort  (cost=690.69..690.70 rows=1 width=353)
         Sort Key: round DESC, tx_index, ((body)::text)
         ->  Index Scan using ix_runtime_events_nft_transfers on runtime_events  (cost=0.42..690.68 rows=1 width=353)
               Index Cond: ((runtime = 'sapphire'::runtime) AND (((body -> 'topics'::text) ->> 3) = 'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAOQ='::text))
               Filter: ((body ->> 'address'::text) = encode(eth_preimage(('oasis1qpwe6h0hw6s0x4skujhznld065300p83wgxnvu0n'::text)::oasis_addr), 'base64'::text))
```

Reply:
gonna resolve, it seemed to work